setup.cfg

```diff
@@ -40,3 +40,4 @@ install_requires =
 [options.extras_require]
 devtools =
     pre-commit~=2.2
+    yapf==0.32.0
```

Comment:
Why do you need to get yapf? In principle the `pre-commit` just installs it in its own environment when running. Is this because you need it in the environment for some auto-fixing feature of vscode?

Reply:
My environment is set to autoformat on save. Best it autoformats using the same formatter version as the pre-commit hook. Do you see an issue with adding this devtools dep?

Reply:
Personally I wouldn't add dependencies that are specific to one devs work environment to the general code, but since in this case you are the only dev of the package I guess it should be ok. The only thing I would be careful about is that now this version here and the one in the pre-commits need to be always in sync. One alternative you could consider to avoid this is what they use in `aiida-core` for `mypy`, [for example](https://github.com/aiidateam/aiida-core/blob/main/.pre-commit-config.yaml#L52-L72), where some of the hooks use the a `local` version of the package (that way you only need to specify the version in the dependencies, and the pre-commit will just use that one).

Reply:
I think this is not crucial. I closed the PR, discarding the `yapf` dependency (I'll just manually install it), and moving the JSON-ignoring commit closer to where it is needed (see #50).